### DISCUSSION TITLE
Please add support for disabling/re-binding trackpad "NUM LK"

### DISCUSSION THREAD

**Discussion by wari-sul** [seerge/g-helper#1078] [Ideas]
Hello, there! Thank you for developing this awesome software. Finally I managed to get rid off asus bloatwares. So, far my experience on my Asus G513QM has been fabulous. But unfortunate Asus NUM Lk is hindering this fabulous experience. Please add support for disabling/re-binding asus trackpad "NUM LK". I know that I can use Microsoft PowerToys to tackle this issue but to me that is another bloatware.

**Comment by seerge** [comment]
Hello, do I understand that you just want to bind default windows Num Lock key? you can use Sharpkeys for that purpose.

Changing windows key bindings it out of scope for the app

**Reply by wari-sul** [reply]
tried it. but unfortunately it doesn't work. :(
